db/internal_stats.cc

```diff
@@ -482,6 +486,9 @@ const std::unordered_map<std::string, DBPropertyInfo>
         {DB::Properties::kLiveSstFilesSize,
          {false, nullptr, &InternalStats::HandleLiveSstFilesSize, nullptr,
           nullptr}},
+        {DB::Properties::kLiveSstFilesSizeAtTemperature,
+         {false, &InternalStats::HandleLiveSstFilesSizeAtTemperature, nullptr,
```

Comment:
Is it possible to turn it to true? Do we need it to be done inside DB mutex? Iterating all files is not so cheap.

Reply:
make sense. Updated.